recipes/r-semnetdictionaries/meta.yaml

```
@@ -0,0 +1,73 @@
+{% set version = '0.1.6' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-semnetdictionaries
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/SemNetDictionaries_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/SemNetDictionaries/SemNetDictionaries_{{ version }}.tar.gz
+  sha256: 1941ab96419b9488d7c90be60fc440d815326d1435265fd2a0127982d9a2cce6
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{posix}}zip               # [win]
+  host:
+    - r-base
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('SemNetDictionaries')"           # [not win]
+    - "\"%R%\" -e \"library('SemNetDictionaries')\""  # [win]
+
+about:
+  home: https://github.com/AlexChristensen/SemNetDictionaries
+  license: GPL (>= 3.0)
```

```suggestion
  license: GPL-3.0-or-later
```

According to:
* https://cran.r-project.org/web/packages/SemNetDictionaries/index.html
* https://spdx.org/licenses/GPL-3.0-or-later.html